### PR TITLE
[15.0][MIG] base_delivery_carrier_label: Migration to 15.0

### PR DESCRIPTION
**Pull request by fshah-initos**
Dependent module: delivery_carrier_info 
Pr: https://github.com/OCA/delivery-carrier/pull/475

### CODE REVIEW

**Comment by fshah-initos** [comment]
Test case and runboat/build error were not solved because the dependent module is not merged in main branch 15.0.

**Comment by florian-dacosta** [comment]
Hello, same problem regarding the commit history here, please follow the method described here : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

**Review by dsolanki-initos — COMMENTED** [review]
Code review and functional test.
Looks good to me and except depended module failed case

**Comment by ikapasi-initos** [comment]
Hi, @sebalix can you please review this pull..?

**Comment by ikapasi-initos** [comment]
Hi, @CarlosRoca13 can you please review this pull..?

**Review by CarlosRoca13 — APPROVED** [review]
Please separate the initial pre-commit changes from the migration commit.

**Review by sebalix — COMMENTED** [review]
Hi, could you integrate last commits from 14.0 branch? There are missing commits for version 14.0.1.2.2 and 14.0.1.3.0 (see https://github.com/OCA/delivery-carrier/commits/14.0/base_delivery_carrier_label )

**Comment by ikapasi-initos** [comment]
> Hi, could you integrate last commits from 14.0 branch? There are missing commits for version 14.0.1.2.2 and 14.0.1.3.0 (see https://github.com/OCA/delivery-carrier/commits/14.0/base_delivery_carrier_label )

Hi, @sebalix I've added missing commits for versions 14.0.1.2.2 and 14.0.1.3.0.
Can you please review this pull..?

**Review by dsolanki-initos — APPROVED** [review]

**Comment by ikapasi-initos** [comment]
@chienandalu can you please review this pull..?

**Comment by chienandalu** [comment]
> @chienandalu can you please review this pull..?

I'm afraid we don't work with that family of modules. Better ask to the usual mantainers.

**Comment by ikapasi-initos** [comment]
@ivantodorovich can you please review this pull..?

File: base_delivery_carrier_label/models/delivery_carrier.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class DeliveryCarrier(models.Model):
     )
 
     def alternative_send_shipping(self, pickings):
-        return [{"exact_price": False, "tracking_number": False}]
+        return {}
```
Comment (rousseldenis) [inline base_delivery_carrier_label/models/delivery_carrier.py]: @fshah-initos Why removing this ?

Comment (ikapasi-initos) [reply base_delivery_carrier_label/models/delivery_carrier.py]: @rousseldenis I've removed it for testing purposes now I added it.

**Comment by dsolanki-initos** [comment]
Hello @gurneyalex, This PR is approved by @OCA-git-bot, Could you please review and merge this PR?

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by jans23** [comment]
@rousseldenis This PR has been approved. Anything we can do to get it merged?

**Pull request merged**